static/templates/settings/settings_save_discard_widget.hbs

```diff
@@ -1,17 +1,17 @@
 <div class="save-button-controls hide">
     <div class="inline-block organization-submission subsection-changes-save">
-        <div class="icon-button button primary save-button" type="button" id="org-submit-{{section_name}}" data-status="save">
-            <span class="fa fa-spinner fa-spin icon-button-loading"></span>
-            <span class="fa fa-check icon-button-icon"></span>
-            <span class="icon-button-text">
+        <div class="custom-settings-button button primary save-button" type="button" id="org-submit-{{section_name}}" data-status="save">
```

Comment:
Might be better to call it save-discard-widget-button and save-discard-widget-text, just to make the names align with the template name?

Reply:
Yeah, sounds better.